lib/AlipayAccount.php

```diff
@@ -7,7 +7,60 @@
  *
  * @package Stripe
  */
-class AlipayAccount extends ExternalAccount
+class AlipayAccount extends ApiResource
```

Comment:
Where are we on the idea that we should remove those classes entirely since we don't support them anymore and have not for over a year? Since PHP is dynamic and would return an `ApiResource` anyway when you get one of those on a Charge for example, is it time to get rid of it at the same time?

Reply:
I think we can keep the classes around -- the definitions do still exist in the OpenAPI spec, and in the case of Bitcoin some users do still send requests to the top-level endpoint `/v1/bitcoin/receivers`.

I've removed the `refund()` method from `BitcoinReceiver` though, and updated the PHPDoc to include links to the sources API documentation.